src/Tools-CodeNavigation/CNSelectorExtractor.class.st

```diff
@@ -53,9 +53,12 @@ CNSelectorExtractor >> extractSelectorFromAST: ast atPosition: aPosition [
 	   self foo:|#bar => Here a user may want foo: or bar.
 	For now we decided to favor foo: to motivate people to indent code correctly"
 	| offset position bestNodeAtPosition |
-	offset := (ast sourceCode at: aPosition - 1) isSeparator
```

Comment:
Why not `(aPosition - 1 max: 0)`?

Reply:
max: 0 would fail, because it would ask for at: 0 in a 1 based context.
I though max: 1 would be good, but actually it finds a letter, which is not a separator, set offset to -1, and fails later in the code.

Reply:
(And I did this to fix the problem in the simplest way possible, didn't even try to do something pretty)

Reply:
Well, there is no need to rush :).

I'd prefer a solution with `or:` which would read better and make it only two branches explicitly.

`(aPosition = 1 or: [(ast sourceCode at: aPosition - 1) isSeparator]) ...`

Reply:
No need to rush a merge indeed! 
I agree, I like your way better, and it's commited :)